src/form/components/FieldAutocompleteAddress.tsx

```diff
@@ -53,6 +71,14 @@ export const FieldAutocompleteAddress = <FormData extends {}>(
     style: props.style,
   }
 
+  // Address search params
+  const [searchParams, setSearchParams] = useState({
+    ZipCode: '',
```

Comment:
The Capitol case code style is a bit against the standard of javaScript but well... let's don't be picky, will we 🤣